layouts/fastly/helix.vcl

```diff
@@ -1000,6 +1006,13 @@ sub vcl_deliver {
 sub vcl_error {
   set req.http.X-Trace = req.http.X-Trace + "; vcl_error";
 #FASTLY error
+
+  if (obj.status == 301) {
```

Comment:
Overriding all 301s with X-Location is perhaps overzealous.
Because this is fairly specific functionality, perhaps error with a unique code (9XX) so this code block only handles what we want and then rewrite the status code.

Reply:
```suggestion
  if (obj.status == 301 && req.http.X-Location) {
```

maybe that's better, @ejthurgo